TotalSegmentator/TotalSegmentator.py

```diff
@@ -272,7 +278,8 @@ def onPackageInfoUpdate(self):
             self.ui.packageInfoTextBrowser.plainText = self.logic.installedTotalSegmentatorPythonPackageInfo().rstrip()
 
     def onPackageUpgrade(self):
-        with slicer.util.tryWithErrorDisplay("Failed to upgrade TotalSegmenter", waitCursor=True):
+        with slicer.util.tryWithErrorDisplay("Failed to upgrade TotalSegmenter", waitCursor=True):           
```

Comment:
Typo - TotalSegmentator (I know it's not a new error, but it's a good time to fix it)